internal/cli/deployment_list.go

```diff
@@ -155,6 +177,30 @@ func (c *DeploymentListCommand) Run(args []string) int {
 				completeTime = humanize.Time(t)
 			}
 
+			// Only display "Latest" report on Last deployment
+			var statusReportComplete string
+			if statusReportResp != nil {
+				if i == 0 { // Latest deployment
+					switch statusReport.Health {
+					case sdk.StatusReport_READY:
+						statusReportComplete = "✔"
+					case sdk.StatusReport_ALIVE:
+						statusReportComplete = "✔"
+					case sdk.StatusReport_DOWN:
+						statusReportComplete = "✖"
+					case sdk.StatusReport_PARTIAL:
+						statusReportComplete = "●"
+					case sdk.StatusReport_UNKNOWN:
+						statusReportComplete = "?"
+					}
+					if t, err := ptypes.Timestamp(statusReport.TimeGenerated); err == nil {
+						statusReportComplete = fmt.Sprintf("%s - %s", statusReportComplete, humanize.Time(t))
```

Comment:
@izaaklauer good call...showing when the report was generated happens here, it's not obvious in code 😄 

![Screenshot from 2021-06-01 08-18-47](https://user-images.githubusercontent.com/810277/120348414-05128880-c2b2-11eb-866e-6ea38bd0bf54.png)

Reply:
Haha i'm sorry, I caught that you were already doing it as I was writing and didn't mean to submit that feedback!

Reply:
Oops! No worries 😄